cloudshell-orch-core/cloudshell/workflow/orchestration/apps_configuration.py

```diff
@@ -49,16 +55,22 @@ def apply_apps_configurations(self, apps):
         apps_configuration = []
 
         for app in apps:
-            if len(app.app_request.appConfiguration) > 0:
-                apps_configuration.append(AppConfiguration(app.deployed_app.Name,
-                                                           app.app_request.appConfiguration))
+            if len(app.app_request.scripts) > 0:
```

Comment:
I think that this can be simplified: `if len(app.app_request.script) > 0` -> `if app.app_request.scripts`

Reply:
same regarding explicit code.